### ISSUE TITLE
No possibility to delete device-specific files?

### ISSUE THREAD

**Issue by mari8b** [BurgerZ/jBART#130]
The structure of 'deletefiles.conf' has changed. The possibility of deleting device-specific files is gone?

**Comment by BurgerZ** [comment]
Yes, but I'll update jbart soon with device-specific configs, like nxt patches.
